### DISCUSSION TITLE
springboot/logback

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#164] [Announcements]
# springboot/logback

关于 Logback 日志系统是一个线上项目必备的素质之一，代表性的日志框架 Log4j、SLF4J、Logback 这哥仨竟然是亲兄弟，他们有一个亲爹，那就是巨佬 Ceki Gulcu。 由于 Spring Boot 的默认日志框架选用的 Logback，再加上 Log4j2 之前爆过严重的漏洞，所以我们这次就只关注 Logback。 1）Logback 非常自然地实现了 SLF4J，不需要像 Log4j 和 JUL 那样加一个适配层。

https://tobebetterjavaer.com/springboot/logback.html

**Comment by qinple** [comment]
希望二哥出一个"整合log4j2"
